libs/net/services/Actions/IngestAction.cs

```diff
@@ -112,13 +120,21 @@ public IngestAction(IApiService api, IOptions<TOptions> options, ILogger<IngestA
             reference.Status != (int)WorkflowStatus.Received &&
             reference.Status != (int)WorkflowStatus.Imported)
         {
-            reference = await this.UpdateContentReferenceAsync(reference, WorkflowStatus.Received);
-            if (reference != null && manager.Ingest.PostToKafka() && content != null)
+            try
+            {
+                reference = await this.UpdateContentReferenceAsync(reference, WorkflowStatus.Received);
+                if (reference != null && manager.Ingest.PostToKafka() && content != null)
+                {
+                    var result = await this.Api.SendMessageAsync(manager.Ingest.Topic, content) ?? throw new InvalidOperationException($"Failed to receive result from Kafka for {reference.Source}:{reference.Uid}");
+                    reference.Offset = result.Offset;
+                    reference.Partition = result.Partition;
+                    reference = await Api.UpdateContentReferenceKafkaAsync(reference, Headers);
+                }
+            }
+            catch (Exception e)
             {
-                var result = await this.Api.SendMessageAsync(manager.Ingest.Topic, content) ?? throw new InvalidOperationException($"Failed to receive result from Kafka for {reference.Source}:{reference.Uid}");
-                reference.Offset = result.Offset;
-                reference.Partition = result.Partition;
-                reference = await Api.UpdateContentReferenceKafkaAsync(reference, Headers);
+                Logger.LogError(e, "Failed in {class}.{method}", nameof(IngestAction<TOptions>), nameof(ContentReceivedAsync));
```

Comment:
You are now triple logging the same error.  It would log in your code change above line 102, then it will log here line 136, and it will log where it already logs in each ingestion service.  What other information are you attempting to find here?

Reply:
Updated, thanks